### PR TITLE
Update Perspective.py

### PR DESCRIPTION
**Pull request by Prabhnith**
removed Java style comments and changes in documentation of the example

### CODE REVIEW

File: examples.py/3D/Camera/Perspective.py [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
- * distant objects appear smaller than closer ones. The parameters
- * define a viewing volume with the shape of truncated pyramid.
- * Objects near to the front of the volume appear their actual size,
- * while farther objects appear smaller. This projection simulates
- * the perspective of the world more accurately than orthographic projection.
- * The version of perspective without parameters sets the default
- * perspective and the version with four parameters allows the programmer
- * to set the area precisely.
- """
+Perspective.
```
Comment (GoToLoop) [inline examples.py/3D/Camera/Perspective.py]: In most of the comments below, 1 trailing space was added.  O_o

File: examples.py/3D/Camera/Perspective.py [hunk 2]
```diff
@@ -28,7 +29,6 @@ def draw():
         aspect = aspect / 2.0
 
     perspective(fov, aspect, cameraZ / 10.0, cameraZ * 10.0)
-
     translate(width / 2 + 30, height / 2, 0)
     rotateX(-PI / 6)
     rotateY(PI / 3 + mouseY / float(height) * PI)
```
Comment (GoToLoop) [inline examples.py/3D/Camera/Perspective.py]: Function **float()** is redundant. Constant THIRD_PI is `PI / 3`.

**Comment by Prabhnith** [comment]
any suggestions

**Comment by jdf** [comment]
Thank you.

**Pull request merged**